docs/content/developers/ontwerpkeuzes-informatiemodellen.md

```diff
@@ -1,2 +1,15 @@
-Bij het ontwikkelen van de API's aam de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie hier) als consequentie hebbe dat een informatiemodel aangpast moet worden. 
+## Aanpassingen RGBZ en ImZTC
+
+Bij het ontwikkelen van de API's aan de hand van user-story's kan voortschrijdend inzicht er toe leiden dat in de API cq. resource afgeweken wordt van het desbetreffende informatiemodel, i.c. RGBZ of ImZTC. Het desbetreffende informatiemodel moet daarop aangepast worden. Ook kan een (generieke) ontwerpkeuze (zie [hier](https://github.com/VNG-Realisatie/gemma-zaken/blob/master/docs/content/developers/design-keuzes.md)) als consequentie hebben dat een informatiemodel aangepast moet worden. 
 Hieronder vermelden we de gemaakte keuzes tot aanpassing van RGBZ en ImZTC. 
+
+[nog uit te werken:]
+- Einddatum van zaak als afgeleid gegeven (en STATUSTYPE in ImZTC met hoogste volgnummer is laatste status). 
+- Omschrijving- en Toelichting-attributen niet meer AN1000 maar AN ('string'); ook BESLUIT.Publicatietekst
+- Richting van relatie ZAAK - INFORMATIEOBJECT omdraaien
+- Richting van relatie BESLUIT - INFORMATIEOBJECT omdraaien 
+- Formele historie op relaties van BESLUIT: ze kunnen gewijzigd worden
+- BESLUIT kan vastgelegd zijn als: regel dat het info-object aan dezelfde zaak gekoppeld moet zijn als het Besluit (nog niet in API)
+- BESLUITTYPE: referentielijsten bij Categorie en Generieke omschrijving (nog niet in API)
```

Comment:
enig idee hoe dit eruit gaat zien/te refereren zal zijn? makkelijkst is als dit via een (publieke, open data) API kan ontsloten worden.

Reply:
@sergei-maertens Goed gezien! Referentielijsten dienen inderdaad met één landelijk aangeboden API ondersteund te worden. Geen idee hoe, maar daar heb jij wellicht ideeen bij. Ik spreek je daarover binnenkort graag 's. Moet hiervan een userstory komen?

Reply:
Tjah, valt buiten onze scope vrees ik, maar iemand moet het inderdaad gaan doen. Een API daarvoor is wel snel uit de grond gestampt hoor. Denk dat dit meer een vraag is voor VNG-Realisatie als beheersorganisatie, misschien @wishalg hierbij betrekken om dit in gang te zetten?

Reply:
Er is ondertussen ook zoiets opgezet voor communicatiekanalen: https://ref.tst.vng.cloud/referentielijsten/api/v1/schema/